aws.yaml.example

```diff
@@ -6,13 +6,22 @@ st2_user_data: /opt/stackstorm/packs/aws/actions/scripts/bootstrap_user.sh
 debug: true
 
 service_notifications_sensor:
-  host: localhost
-  port: 12345
-  path: /my-path
+  - host: localhost
+  - port: 12345
+  - path: /my-path
 
 sqs_sensor:
+  roles_arns:
+    - arn:aws:iam::123456789098:role/rolename1
+    - arn:aws:iam::901234567812:role/rolename2
+    - arn:aws:iam::567890123489:role/rolename3
   input_queues:
-    - queuename
+    - queue_name_1
+    - https://sqs.us-east-1.amazonaws.com/345678901223/queue_name_2
```

Comment:
Poate n-ar trebui pus asta cu account care nu exista in roles_arns. E totusi un exemplu, ar trebui sa fie corect.